modules/cli/apply/apply.go

```diff
@@ -181,48 +181,42 @@ func save(client *client.Getter, bundle *fleet.Bundle, imageScans ...*fleet.Imag
 
 	obj, err := c.Fleet.Bundle().Get(bundle.Namespace, bundle.Name, metav1.GetOptions{})
 	if apierrors.IsNotFound(err) {
-		_, err = c.Fleet.Bundle().Create(bundle)
-		if err == nil {
-			fmt.Printf("%s/%s\n", bundle.Namespace, bundle.Name)
-		} else {
+		if _, err = c.Fleet.Bundle().Create(bundle); err != nil {
 			return err
 		}
+		fmt.Printf("created: %s/%s\n", bundle.Namespace, bundle.Name)
```

Comment:
Use `logurs` here and other instances of `fmt.Printf`.

Reply:
This was refactoring the original `fmt.Printf` statements, so I missed this. Good call.